### PR TITLE
Refactor docs for open source friendliness

### PR DESCRIPTION
**Pull request by dperconti**
- Add Changelog doc
- Add Contributing doc
- Update Readme with reference to new files
- Update setup and conf with lululemon branding

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -19,22 +19,19 @@
...
 setup(
     name='tight',
     version='0.1.0',
     description="Microframework",
     long_description=README,
-    author="Michael McManus",
-    author_email='michaeltightmcmanus@gmail.com',
-    url='https://github.com/michaelorionmcmanus/tight',
+    author="lululemon athletica",
+    url='https://github.com/lululemon/tight',
```
Comment (onewheelskyward) [inline setup.py]: Did you get that opensource@lulu email working?

**Comment by onewheelskyward** [comment]
If we have the email let's add it, otherwise I'll mergebot.  Solid work.

File: setup.py (hunk 1 above)
Comment (dperconti) [reply setup.py]: I've gotten a confirmation that it's been created, but I've been testing it and I'm still not receiving any of the emails.

Comment (onewheelskyward) [reply setup.py]: hah, that figures.  Good on ya.

**Pull request merged**